ra2ce/network/network_wrappers/vector_network_wrapper.py

```diff
@@ -111,6 +114,17 @@ def get_network(
         edges_complex, _ = nut.graph_to_gdf(graph_complex)
         logging.info("Finished converting the graph to a geodataframe")
 
+        # Segment the complex graph
+        if not math.isnan(self.segmentation_length):
```

Comment:
This is repetition from what happens in OSMNetworkWrapper. Make it a function and use it there

Reply:
Ok, I make a function at the protocol level.